src/proto/grpc/testing/messages.proto

```diff
@@ -135,7 +140,7 @@ message StreamingOutputCallRequest {
   Payload payload = 3;
 
   // Whether to request the server to compress the response.
-  bool request_compressed_response = 6;
+  bool response_compressed = 6;
```

Comment:
Putting 
"BoolValue compressed" field to ReponseParameters seems to be  the right thing to do.